examples/media/src/widgets/Articles.ts

```diff
@@ -0,0 +1,151 @@
+import instantsearch from 'instantsearch.js';
+import { connectInfiniteHits } from 'instantsearch.js/es/connectors';
+import { distanceInWords } from 'date-fns';
+import { getCurrentDate } from '../utils';
+
+const currentDate = getCurrentDate();
+
+function createHit(hit, { isHighlighted }) {
+  return `
+<li class="ais-InfiniteHits-item${
+    isHighlighted ? ' infinite-hits-item--highlighted' : ''
+  }">
+  <article class="card">
```

Comment:
This could looks better with placeholders to minimize motion on image load.
https://css-tricks.com/building-skeleton-screens-css-custom-properties/

Reply:
Definitely! I'll work on this to make progress on researching how to better handle low-network connections with InstantSearch.

Reply:
I added a basic implementation in 51c6f413f29e19d8d474011ed1a38e6632d36fbb.